index.rst

```diff
@@ -44,14 +44,579 @@
+
+This working group also created a proof-of-concept of the critcal implementation recommendations and found them to be successful in satisfying the requirements and being relatively straightforward to implement.
+Details are found in the `Proof-of-concept Demonstrations`_ section.
+
+One should also note that there were functionalities that that group found to be critical to the success of commissioning, but not directly for on-the-fly applications, which therefore resulted in the requirement being out-of-scope. 
+These types of issues are a grouped into the `Other Findings and Identified Issues`_ and should be strongly considered for implementation as part of the change requests that will result from this charge.
+
+
+Augmenting Current Functionality
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When considering how to implement the use-cases, effort was always made to ensure that currently available tools (presented in `Deliverable 3`_) would be used wherever appropriate.
+In most cases, specifically in regards to image display, augmenting functionality of existing tools is a perferred path to starting from scratch.
+
+The list of new functionalities required for already existing tools include:
+
+#. Numerous `Camera Visualization Improvements <pg-D2-Requirements_for_image_display>`_ were described as part of `Deliverable 2`_ and are therefore not repeated here.
+   An `example of the callback functionality <demo_callback>`_ in the `Proof-of-concept Demonstrations`_ section.
+#. The OCPS (really the butler) requires access to EFD. This is not currently captured in a use-case but one can envision how having a pipeTask be capable to correlate image quality against items in the EFD could be useful.
+
+   - No code has been written to integrate butler directly with EFD, but it is possible to do
+   - Would enable useres to define pipelines that explicitly specified EFD datasets as pipeline inputs. 
+     Currently, it would be required to sort out the mapping of Exposure dataId to and EFD call in (potentially) a special runQuantum method in the pipeline task
+  
+
+Entirely New Functionality
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section identifies functionalities that are required and could not be assertained by upgrading already existing components.
+The largest piece of missing functionalty is the framework to perform on-the-fly analyses which are triggered on specific events or conditions, then able to perform calculations, generate a report (including plots etc), and have the operator be alerted.
+Implementing this type of capability requires numerous pieces to work together.
+
+
+Catcher CSC
+'''''''''''''
+
+A series of new functionality, which for the purposes of this document we have grouped into a single "Catcher" Controllable SAL Component (CSC) is required to handle the low-level coordiation of identifying when a specific condition is met, then launching and monitoring an analysis process.
```

Comment:
"CSC" appears several times before this instance. The acronym is defined here for the first time.